packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -16,6 +16,11 @@ import 'text_span.dart';
 
 export 'package:flutter/services.dart' show TextRange, TextSelection;
 
+// The default font size if none is specified. This should be kept in
+// sync with the default values in text_style.dart, as well as the
+// defaults set in the engine (eg, LibTxt's text_style.h, paragraph_style.h).
```

Comment:
These are not formal API docs, so I included a reference to LibTxt, I can remove if people feel it is best not to link to specific engine implementation that is slated to be swapped out.

Reply:
I trust you to update this comment once libtxt is swapped out :)

Reply:
If you feel like an overachiever though, you could consider moving _kDefaultFontSize to a shared constants file in painting/constants.dart.  We have a few files like that elsewhere but none in painting yet.